package.json

```
@@ -0,0 +1,13 @@
+{
+    "name": "password-manager-resources",
```

Should probably mark this as private, yeah?
```suggestion
    "name": "password-manager-resources",
    "private": true,
```

---

Sure, up to you guys.